frontend/website-redesign/src/pages/Community.re

```diff
@@ -225,7 +225,7 @@ let make = (~profiles) => {
     </div>
     <QuoteSection
       small=false
-      copy={js|"My measure of a project isn't the quality of the tech. It’s the quality of the community. I wouldn't have been able to spin my node up if it weren't for the insanely great members that helped me. And that's something special.”|js}
+      copy={js|My measure of a project isn't the quality of the tech. It’s the quality of the community. I wouldn't have been able to spin my node up if it weren't for the insanely great members that helped me. And that's something special.”|js}
```

Comment:
If this is a quote section, shouldn't it have a beginning quote? 

EDIT: I checked the build and the quote seems to be there so now I'm just confused

Reply:
hahahah the quote is literally absolutely positioned separately bc umi wants it "to be hanging out of the [container]"

Reply:
yikes

Reply:
thanks for flagging! ill fix it in a follow up pr since this is getting kinda big and im going to show this to kate shortly

Reply:
Could you revert it back if that's the case? I think the quote being slightly out of place by design is better than the mobile version not working.